db/demo_seeds/legislation_processes.rb

```diff
@@ -1,480 +1,861 @@
-      non-discrimination of the Roma population" of the Government Action Plan; channels for
-      participation in municipal policies that affect them by entities and persons belonging to
-      groups that are more susceptible to racial discrimination.</p>\r\n
-    DESCRIPTION
-    tag_list: "Education,Health",
-    author_id: author.id,
-    responsible_name: author.username,
-    terms_of_service: "1",
-    created_at: 1.week.ago
-  )
-
-  supporters = users.sample(5)
-  supporters.each do |supporter|
-    proposal.vote_by(voter: supporter, vote: true)
+  section "Legislation Process 4" do
+    #
+    # Legislation Process 4
+    #
+    process = Legislation::Process.create!(
+      title: "Strategic Mobility Plan 2025-2035",
+      summary: "The Strategic Mobility Plan 2025-2035 will be written together with our citizens.\r\n",
+      description: "The Strategic Mobility Plan 2025-2035 will be written" \
+                   " together with our citizens. You can participate in the" \
+                   " drafting of the Plan during three consecutive phases:\r\n" \
+                   "1. Join the discussion about what are the most important" \
+                   " mobility themes for the city\r\n" \
+                   "2. Submit proposals or ideas for Mobility projects or regulations in the city\r\n" \
+                   "3. Comment on the first and second draft of the Strategic Mobility Plan 2025-2035\r\n",
+      homepage: <<~HOMEPAGE,
+        `<p>Participation in the city has a few rules and guidelines. These are:</p>
+         <ol>
+         <li>What happens with my input? Your input will be used to write up the Strategic
+         Mobility Plan 2025-2035, as long as the suggestions, proposals and comments are feasible
+         and fit within the wider Mobility Framework of the city. Participants will receive structural
+         feedback before and after each phase of the process.</li>
+         <li>What is the Strategic Mobility Plan about? The Strategic Mobility Plan 2025-2035 will
+         define the priorities and policy choices of the city for the next ten years. It will define
+         sustainability variables and projects, determine which modes of transport and which
+         districts of the city get prioritized, and it will set a timeline for completion of projects</li>
+         <li>Are there any rules for participation? Yes, we carefully enforce certain rules
+         for participation</li>
+         <li>Collaboration. It is important to look at other people's contribution. We will look at
+         everything but will eventually accept proposals with the most support and/or votes.
+         2) Be concise. We value short and concise contributions, which will positively affect the
+         chances of proposals being integrated in the Strategic Mobility Plan 2025-2035.
+         3) No discrimination. Offensive languages will not be tolerated and comments.</li>
```

Comment:
Just a small note on this list item `<li>`

Just noticed this list starts with "Collaboration." but then goes to "2)" and "3)".
Should we add a "1)" to keep it consistent? 🤔